content/sql/concepts/math-functions/math-functions.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Math Functions'
+Description: 'SQL Math Functions are built-in features in SQL, which let you do mathematical operations, directly in your database queries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+**SQL Math Functions** are built-in features in SQL, which let you do mathematical operations, such as addition, subtraction, and even more complex calculations, directly in your database queries. This makes it easier and more efficient to work with numerical data in SQL, as you don't need to extract the data and process it separately.
+
+### Example
+
+In this example, the following data is given, in a table named `orders`:
+
+| order_id | product_price | quantity |
+| -------- | ------------- | -------- |
+| 1        | 20            | 3        |
+| 2        | 15            | 5        |
+| 3        | 30            | 2        |
+| 4        | 10            | 1        |
+
+The multiplication (\*) math function can be used to calculate the total price for each order:
```

Comment:
Multiplication (and addition, subtraction and division) is an operator, we'd like to see an example with an actual function. such as `ABS()`, `FLOOR()` or `CELING()`